pkg/config/config.go

```diff
@@ -107,6 +107,8 @@ var (
 	redfishSkipSSLVerify          = getBoolConfig("REDFISH_SKIP_SSL_VERIFY", true)
 
 	////////////////////////////////////
+	MachineIDKey        = "MACHINE_ID"
```

Comment:
does it make more sense to rename it is `MACHINE_SPEC_ID` ?

Reply:
Removed.